submissions/vincentNjele/Database/accident_detail.sql

```diff
@@ -0,0 +1,146 @@
+
+
+
+/* 1. Number of people who survived   */
+SELECT  count(s.survival_status) FROM survival_table s WHERE s.survival_status = 1
+
+/*Number of survivors = 290   */
+
+
+/* 2. Number of people who did not survived   */
+SELECT  COUNT(s.survival_status) FROM survival_table s WHERE s.survival_status = 0
+
+/*Number of passengers that did not survive = 424   */
+
+
+/*3. Those who were <27 and embarked at Queenstown, I could not get those that embarked and Q and C in the same query because they are all in the same column  */
```

Comment:
Its in the same column. yes but it is possible to get those that embarked from Q and C